packages/react-native-platform-override/src/Hash.ts

```diff
@@ -64,12 +65,17 @@ export async function hashFileOrDirectory(
     return hashContent((await repo.readFile(name))!);
   } else {
     const hasher = new Hasher();
-    for (const file of (await repo.listFiles([`${name}/**`])).sort()) {
+    const subfiles = await repo.listFiles([`${unixPath(name)}/**`]);
+
+    for (const file of subfiles.sort()) {
       const contents = await repo.readFile(file);
       hasher.feedContent(contents!);
 
       // Incorporate the filename to detect if renames happen
-      hasher.feedContent(path.relative(name, file));
+      const platformIndependentPath = unixPath(
+        path.relative(name, normalizePath(file)),
```

Comment:
Don't think we need the second normalize here, since the file is directly from the FileRepository. Would be a bug with the mock if we do.

Reply:
Oh no, this was the case of tests using raw paths + path.relative not understanding Windows slashes on Unix